packages/aws-cdk-lib/aws-iam/README.md

```diff
@@ -369,6 +369,37 @@ new iam.Role(this, 'Role', {
 });
 ```
 
+### Granting assume role permission from a role
+
+Principals can be granted permission to assume a role using `grantAssumeRole`.  Note that this does not apply to Service Principals or Account Principals as they must be added to the role Trust Policy.
```

Comment:
Nit: lowercase `service principals` [1]
Nit: lowercase `account principals [2]

Also move the note to a newline. 

[1] https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-services
[2] https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-accounts